README.md

```diff
@@ -31,6 +31,26 @@ Here is a brief list of the available widgets:
 * **edit_box**: an editable text box (multi-line edit_boxes are not yet fully supported).
 * **scroll_frame**: a frame that has scrollable content.
 
+# Getting started 
+```
+git clone https://github.com/unnamed-mmo/lxgui
```

Comment:
This would have to be change to point to this repo

Reply:
whoops! 😳